compat/src/internal.d.ts

```diff
@@ -1,30 +1,41 @@
 import { Ref } from '../..';
 import {
-  Component as PreactComponent,
-  VNode as PreactVNode,
-  FunctionalComponent as PreactFunctionalComponent
+	Component as PreactComponent,
+	VNode as PreactVNode,
+	FunctionalComponent as PreactFunctionalComponent
 } from '../../src/internal';
+import { SuspenseProps } from './suspense';
 
 export { ComponentChildren } from '../..';
 
 export { PreactElement } from '../../src/internal';
 
 export interface Component<P = {}, S = {}> extends PreactComponent<P, S> {
-  isReactComponent: object;
-  isPureReactComponent?: true;
+	isReactComponent?: object;
+	isPureReactComponent?: true;
+
+	_childDidSuspend?(error: Promise<void>);
 }
 
 export interface FunctionalComponent<P = {}> extends PreactFunctionalComponent<P> {
-  shouldComponentUpdate?(nextProps: Readonly<P>): boolean;
-  _forwarded?: true;
+	shouldComponentUpdate?(nextProps: Readonly<P>): boolean;
+	_forwarded?: true;
 }
 
 export interface VNode<T = any> extends PreactVNode<T> {
-  $$typeof: symbol | string;
-  preactCompatNormalized: boolean;
+	$$typeof?: symbol | string;
```

Comment:
Made these optional cuz they are only on the VNode after the `vnode` option is invoked so technically they are undefined for a moment, but really because it makes VSCode happy lol